users/rossman360/rossman360.h

```diff
@@ -0,0 +1,8 @@
+#ifndef USERSPACE
```

Comment:
This should ideally use the `#pragma once` include guard, instead of `#ifndef`/`#define`/`#endif`.

```suggestion
#pragma once
```